instrument/D22_Parameters.xml

```diff
@@ -14,6 +14,13 @@
       <parameter name="grouping" type="string">
         <value val="0-255,256-511,512-767,768-1023,1024-1279,1280-1535,1536-1791,1792-2047,2048-2303,2304-2559,2560-2815,2816-3071,3072-3327,3328-3583,3584-3839,3840-4095,4096-4351,4352-4607,4608-4863,4864-5119,5120-5375,5376-5631,5632-5887,5888-6143,6144-6399,6400-6655,6656-6911,6912-7167,7168-7423,7424-7679,7680-7935,7936-8191,8192-8447,8448-8703,8704-8959,8960-9215,9216-9471,9472-9727,9728-9983,9984-10239,10240-10495,10496-10751,10752-11007,11008-11263,11264-11519,11520-11775,11776-12031,12032-12287,12288-12543,12544-12799,12800-13055,13056-13311,13312-13567,13568-13823,13824-14079,14080-14335,14336-14591,14592-14847,14848-15103,15104-15359,15360-15615,15616-15871,15872-16127,16128-16383,16384-16639,16640-16895,16896-17151,17152-17407,17408-17663,17664-17919,17920-18175,18176-18431,18432-18687,18688-18943,18944-19199,19200-19455,19456-19711,19712-19967,19968-20223,20224-20479,20480-20735,20736-20991,20992-21247,21248-21503,21504-21759,21760-22015,22016-22271,22272-22527,22528-22783,22784-23039,23040-23295,23296-23551,23552-23807,23808-24063,24064-24319,24320-24575,24576-24831,24832-25087,25088-25343,25344-25599,25600-25855,25856-26111,26112-26367,26368-26623,26624-26879,26880-27135,27136-27391,27392-27647,27648-27903,27904-28159,28160-28415,28416-28671,28672-28927,28928-29183,29184-29439,29440-29695,29696-29951,29952-30207,30208-30463,30464-30719,30720-30975,30976-31231,31232-31487,31488-31743,31744-31999,32000-32255,32256-32511,32512-32767" />
       </parameter>
+      <!-- The pixel sizes used by SolidAngle -->
```

Comment:
D22 has also the `lr` version, could you add the parameters also there?
Could you also edit this comment line in all its instances to:
```suggestion
      <!-- The pixel sizes [mm] used by SolidAngle, resolution and default Q binning calculations.
      Note that for the panels, often pixels have transposed shape, but the surface is the same.
      Q binning is based on the maximum of the 2 dimensions, so it is also invariant. -->
```